detectron2/engine/train_loop.py

```diff
@@ -279,20 +285,24 @@ def run_step(self):
             losses = sum(loss_dict.values())
 
         """
-        If you need to accumulate gradients or do something similar, you can
-        wrap the optimizer with your custom `zero_grad()` method.
```

Comment:
As the comment says, gradient accumulation should be implemented with a custom optimizer.

Reply:
Gradient accumulation is a pretty fundamental option. It would be nice to have it available out of the box. PytorchLightning does provide this, is there a reason detectron2 can't?

Reply:
Would you please show a working example? I tried to do this but failed to make it work.